docs/tempo/website/configuration/_index.md

```diff
@@ -15,6 +15,7 @@ This section explains the configuration options for Tempo as well as the details
   - [Compactor](#compactor)
   - [Storage](#storage)
   - [Memberlist](#memberlist)
+  - [Compression](#compression)
 
 ## Authentication/Server
 Tempo uses the Weaveworks/common server.  See [here](https://github.com/weaveworks/common/blob/master/server/server.go#L45) for all configuration options.
```

Comment:
```suggestion
Tempo uses the Weaveworks/common server.  Refer to [this page](https://github.com/weaveworks/common/blob/master/server/server.go#L45) for all configuration options.
```

Reply:
This is not really in the scope of the PR i'm making.  I'd rather make this PR separately if we desire this change.

Reply:
Sure, it is a documentation style issue about consistency, I can take care of it later.